.github/workflows/windows.yml

```diff
@@ -0,0 +1,109 @@
+name: 🪟 Windows
```

Comment:
Can you rename this workflow and related files from `windows`  to `windows-vcpkg`  or similar? Compilation for Windows can be achieved with other package managers (like conda or conan), so marking this as `windows`  may be a bit confusing.

Reply:
I will rename it.

Reply:
done.